src/App.tsx

```diff
@@ -165,38 +182,75 @@ class Board extends React.Component<BoardProps, BoardState> {
         }
     }
 
+    public handleGlobalMouseDown = (e: MouseEvent) => {
+        if (!e.defaultPrevented) {
```

Comment:
if preventDefault is called by earlier on then it shouldn't log anything. Currently works with the Square Mouse click events but not with the "Normal" control button that I'm testing. Going to later implement what i do with the normal control button to all other buttons.